Components/Experience.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import EducationList from "./EducationList";
+import EducationListHolder from "./EducationListHolder";
+import GiftComponents from "./GiftComponents";
```

Comment:
Separate the external imports from internal ones. All external imports(packages) should go at the top of the page, then an empty line followed by all internal imports such as components, pages, etc.

```javascript
import React from "react";
...all external imports

import EducationList from "./EducationList";
... all internal imports

```